### PR TITLE
Fix blobstore deploy bug. Image upload now works on prod and on test server.

### PR DESCRIPTION
**Pull request by juanuribe28**

### CODE REVIEW

File: portfolio/src/main/webapp/scripts/comments.js [hunk 1]
```diff
@@ -114,7 +114,7 @@ function scoreToColor(score) {
  */
 function scoreToIndex(score) {
   let index = Math.floor((score+1)*2.5);
-  index = min(4, index);
+  index = Math.min(4, index);
```
Comment (daperlmut) [inline portfolio/src/main/webapp/scripts/comments.js:117]: [Optional] This should probably be part of a separate PR but since it's so small I think it's ok to leave it in here if you don't want the hassle of splitting it out.

Comment (juanuribe28) [reply portfolio/src/main/webapp/scripts/comments.js:117]: Ok, thanks
